governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals/ProposalRow.tsx

```diff
@@ -55,15 +55,13 @@ export const ProposalRow = ({
 
         // set proposal time
         if (!time) {
-          connection
-            .getConfirmedSignaturesForAddress2(proposal.publicKey)
-            .then((txs) => {
-              if (isCancelled) return
-              const firstBlockTime = txs?.[txs.length - 1]?.blockTime
-              if (firstBlockTime) {
-                setTime(new Date(firstBlockTime * 1000))
-              }
-            })
+          connection.getSignaturesForAddress(proposal.publicKey).then((txs) => {
+            if (isCancelled) return
+            const firstBlockTime = txs?.[txs.length - 1]?.blockTime
+            if (firstBlockTime) {
+              setTime(new Date(firstBlockTime * 1000))
+            }
+          })
```

Comment:
We should probably add a `catch`, even if it's just to log -- generally you should include a `catch` any time you use a promise without async/await, otherwise uncaught async errors can put the application in a bad state but be very hard to debug

Reply:
done